cypress/e2e/Intro_Presence_Check.cy.js

```diff
@@ -0,0 +1,11 @@
+describe("template spec", () => {
+	it("passes", () => {
+		cy.visit("https://acm.cs.ucr.edu");
```

Comment:
this will run a production test, but we need to make sure it is for development, change the link to be localhost:3000